### PR TITLE
[17.0][UPD] copier version 1.20

### PR DESCRIPTION
**Pull request by MohamedOsman7**

### CODE REVIEW

**Comment by legalsylvain** [comment]
Did you run copier update ? no files changed...

File: base_view_inheritance_extension/__manifest__.py [hunk 1]
```diff
@@ -11,6 +11,6 @@
     "summary": "Adds more operators for view inheritance",
     "website": "https://github.com/OCA/server-tools",
     "depends": ["base"],
-    "external_dependencies": {"python": ["astor"]},
+    "external_dependencies": {"python": ["astor", "lxml<5.2.0"]},
```
Comment (legalsylvain) [inline base_view_inheritance_extension/__manifest__.py]: Why this change ?

Comment (MohamedOsman7) [reply base_view_inheritance_extension/__manifest__.py]: @legalsylvain i fixed it

**Comment by gurneyalex** [comment]
updated to 1.23
